src/main/scala/cromwell/binding/expression/WdlStandardLibraryFunctions.scala

```diff
@@ -9,25 +9,78 @@ import scala.util.{Failure, Success, Try}
 trait WdlStandardLibraryFunctions extends WdlFunctions[WdlValue] {
   private def fail(name: String) = Failure(new UnsupportedOperationException(s"$name() not implemented yet"))
 
+  private def extractObjectOrArray(params: Seq[Try[WdlValue]]): Try[Array[WdlObject]] = for {
```

Comment:
How about calling this `extractObjects` since it signature returns an array of `WdlObject`?  Callers can determine for themselves whether > 1 object is okay.